response_operations_ui/controllers/uaa_controller.py

```diff
@@ -432,13 +432,12 @@ def get_users_list(
     logger.info("Attempting to fetch user records")
     url = f"{app.config['UAA_SERVICE_URL']}/Users"
     response = requests.get(url, params=param, headers=headers)
-    # TODO Update this function to only have one return type (dict).  Multiple return types isn't great.
     try:
         response.raise_for_status()
         return response.json()
     except HTTPError:
         logger.error("Unauthorised attempt to get user list.", status_code=response.status_code)
-        raise "Unauthorised Access"
+        raise {"error": "Unauthorised Access", "totalResults": 0, "resources": []}
```

Comment:
This is raising a dict.  I didn't know you could and I'm surprised this is working >_<  This should probably be replaced with return?